snap/snapcraft.yaml

```diff
@@ -3,28 +3,53 @@ version-script: git tag -l | sort --version-sort | tail -n1 | cut -c 2-
 version: git
 summary: A command line tool for DigitalOcean services
 description: doctl is a command line tool for DigitalOcean services using the API.
+base: core18
 
-grade: stable
+grade: devel
 confinement: strict
 
 build-packages:
   - gcc
   - git
 
 parts:
-  go:
-    source-tag: go1.11.4
+  launcher:
+    source: snap/local
+    plugin: dump
+    organize:
+      '*': bin/
+    override-stage: |
+      cd $SNAPCRAFT_PART_INSTALL
+      chmod +x bin/doctl-launch
+      snapcraftctl stage
   doctl:
-    after: [go]
+    after: [launcher]
     source: .
     plugin: go
     go-importpath: github.com/digitalocean/doctl
     go-packages: [github.com/digitalocean/doctl/cmd/doctl]
+    organize:
+      bin/doctl: bin/doctl.real
+
+plugs:
+  kube-config:
+    interface: personal-files
+    write:
+      -	$HOME/.kube
```

Comment:
This tab breaks the build for me. Replacing it with a space resolves the issue:

```
Issues while validating snap/snapcraft.yaml: found character that cannot start any token on line 38, column 8
```

Reply:
Gah, thought I had gotten rid of that. Thanks for catching it. Fixed.